### ISSUE TITLE
Create user routes

### ISSUE THREAD

**Issue by DianaLunaXIV** [BrenniusXIV/forum007#12]
-User routes have to include some session validation, and must be hit at creation and at login.

**Comment by DianaLunaXIV** [comment]
These routes look super clean so far - nice work! Only two things to note so far:

1. There is an issue with PUT that has nothing to do with your routes code. I just posted a new issue for it. 
2. We also need a route to get individual user information by id (GET /user/:id). This, in particular, will likely involve pulling the user_id variable from req.session for someone who is logged in. If you want to wait on adding the session stuff, please go ahead and create the route using req.params.id instead.
